jax/_src/scipy/special.py

```diff
@@ -1465,3 +1465,22 @@ def expn_jvp(n, primals, tangents):
 def exp1(x):
   (x,) = _promote_args_inexact("exp1", x)
   return expn(1, x)
+
+
+@_wraps(osp_special.factorial)
```

Comment:
I would use the `lax_description` argument to `_wraps` to mention that `exact` is not implemented.